invokeai/frontend/web/package.json

```diff
@@ -92,7 +97,6 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.1.2",
     "madge": "^6.0.0",
-    "patch-package": "^6.5.1",
```

Comment:
~~Isn't this a dev dependency?~~ why are `patch-package` and `@chakra-ui/cli` needed as normal dependencies?

Reply:
I would love to hear other ideas - but right now the `postinstall` script runs when you install this package, so if those two packages aren't in the bundle it craps out. I actually would love to _not_ run the postinstall script in this use case but can't find a good way to do that. Have you run into this before?

Reply:
Actually I tried forking the unmaintained redux persist - but that project is extremely dead with a nonfunctional main branch and last release build depending on a package with a security issue, which has been wiped from npm. Resulting on it being impossible to build. So patching of some sort is the only option for it afaik

Reply:
Yeah I feel like ideally the invoke-ai-ui package would install and include that step so that the patches are included in the built bundle but then it wouldn't run _again_ when the package was installed as a dependency. We will just stick with --ignore-scripts for now but eventually will need a better solve. Any other issues with this PR @psychedelicious ?